routing/routing_tests/turns_sound_test.cpp

```diff
@@ -106,13 +106,13 @@ UNIT_TEST(TurnsSoundMetersTest)
 
   // Starting nearing the turnItem.
   // 1000 meters till the turn. No sound notifications is required.
-  notificationManager.GenerateTurnNotifications(turns, turnNotifications);
+  notificationManager.GenerateTurnNotifications(turns, turnNotifications, "");
```

Comment:
Would it be easier if a third parameter will be optional, with an empty string by default?

Reply:
Good catch